test/unit_tests/braket/experimental/autoqasm/test_api.py

```diff
@@ -524,15 +526,23 @@ def test_measurement_qubit_discovery(ground_state_measurements) -> None:
 def test_simple_measurement(ground_state_measurements) -> None:
     """Test that a program with only measurements is generated correctly."""
     expected = """OPENQASM 3.0;
+output bit retval_;
```

Comment:
why is this not `bit[3]`?

Reply:
shoot, good catch. I will investigate

Reply:
A solution is not immediately forthcoming. It's not enough to just set `output_parameter.size = size`. The ast generation ignores it or errors, which the approaches I've tried so far